gramex/handlers/modelhandler.py

```diff
@@ -6,16 +6,44 @@
 import gramex.data
 from gramex.handlers import BaseHandler
 import tornado.escape
+from io import BytesIO
+
+_model_cache = {}
 
 
 class ModelHandler(BaseHandler):
     '''
     Allows users to create API endpoints to train/test models exposed through Scikit-Learn.
     TODO: support Scikit-Learn Pipelines for data transformations.
     '''
+    def _load_keras_model(self, path):
+        """Load a Keras model into a local cache.
+
+        Parameters
+        ----------
+        path : str
+            Path to a Keras model.
+        """
+        model = _model_cache.get(path, False)
+        if not model:
+            _model_cache[path] = gramex.cache.open(path)
+        return _model_cache[path]
```

Comment:
@jaidevd -- Since you're using `gramex.cache` you can probably get rid of `_model_cache` another cache check?